### PR TITLE
Add attachable field to network create

### LINKED ISSUE #1336: Support Attachable flag to network create

### PR DESCRIPTION
**Pull request by dnephin**
Fixes #1336

### CODE REVIEW

File: docker/api/network.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def networks(self, names=None, ids=None):
     @minimum_version('1.21')
     def create_network(self, name, driver=None, options=None, ipam=None,
                        check_duplicate=None, internal=False, labels=None,
-                       enable_ipv6=False):
+                       enable_ipv6=False, attachable=False):
```
Comment (shin-) [inline docker/api/network.py]: Maybe we should have `attachable=None` here, and only accept either value (True/False) if API version >= 1.24?

File: docker/api/network.py [hunk 2]
```diff
@@ -91,7 +94,8 @@ def create_network(self, name, driver=None, options=None, ipam=None,
             'Driver': driver,
             'Options': options,
             'IPAM': ipam,
-            'CheckDuplicate': check_duplicate
+            'CheckDuplicate': check_duplicate,
+            'Attachable': attachable,
```
Comment (shin-) [inline docker/api/network.py]: Only add the Attachable field to data if it's != None and API version >= 1.24

**Comment by shin-** [comment]
Couple comments based on my current understanding that this option was introduced with API 1.24.

**Comment by dnephin** [comment]
Only the CS release had it in 1.24 I think, but I guess that's still more correct than no check.

File: docker/api/network.py [hunk 3]
```diff
@@ -116,6 +119,12 @@ def create_network(self, name, driver=None, options=None, ipam=None,
                                      'supported in API version < 1.22')
             data['Internal'] = True
 
+        if attachable is not None
```
Comment (shin-) [inline docker/api/network.py]: Missing a colon

**Comment by shin-** [comment]
Hmm, I don't know what's the cause, but it doesn't look like the `Attachable` option is being registered by my Engine (1.13.0-rc2)

```
In [10]: net_id = c.create_network(name='helloworld', attachable=True)
In [12]: c.inspect_network('helloworld')['Attachable']
Out[12]: False
```

**Pull request closed without merging**